src/test/java/com/inventorsoft/junit/mapper/BookMapperTest.java

```diff
@@ -0,0 +1,149 @@
+        book.setId(ID);
+        book.setTitle(TITLE);
+
+        authorResponse.setBirthDate(AUTHOR_BIRTHDAY);
+        authorResponse.setName(AUTHOR_NAME);
+        authorResponse.setId(AUTHOR_ID);
+
+        bookResponse.setId(ID);
+        bookResponse.setTitle(TITLE);
+        bookResponse.setReleaseDate(RELEASE_DATE);
+        bookResponse.setDescription(DESCRIPTION);
+        bookResponse.setAuthor(authorResponse);
+    }
+
+
+    @Test
+    void mapEntitiesToResponses() {
+        List<Book> list = List.of(book);
+        List<BookResponse> expectedResult = List.of(bookResponse);
+        when(authorMapper.mapEntityToResponse(book.getAuthor())).thenReturn(authorResponse);
+        List<BookResponse> actualResult = bookMapper.mapEntitiesToResponses(list);
+
+
+        assertEquals(expectedResult.size(), actualResult.size());
+        for (int i = 0; i < actualResult.size() && i < expectedResult.size(); i++) {
+            assertEquals(expectedResult.get(i).getId(), actualResult.get(i).getId());
+            assertEquals(expectedResult.get(i).getTitle(), actualResult.get(i).getTitle());
+            assertEquals(expectedResult.get(i).getReleaseDate(), actualResult.get(i).getReleaseDate());
+            assertEquals(expectedResult.get(i).getDescription(), actualResult.get(i).getDescription());
+            assertEquals(expectedResult.get(i).getAuthor(), actualResult.get(i).getAuthor());
+        }
+    }
+
+    @Test
+    void mapEntityToResponse() {
+        when(authorMapper.mapEntityToResponse(book.getAuthor())).thenReturn(authorResponse);
+
+        BookResponse actualResponse = bookMapper.mapEntityToResponse(book);
+
+        assertEquals(bookResponse.getId(), actualResponse.getId());
+        assertEquals(bookResponse.getTitle(), actualResponse.getTitle());
+        assertEquals(bookResponse.getDescription(), actualResponse.getDescription());
+        assertEquals(bookResponse.getReleaseDate(), actualResponse.getReleaseDate());
+        assertEquals(bookResponse.getAuthor(), actualResponse.getAuthor());
+    }
+
+    @Test
+    void mapCreateRequestToEntity() {
+        CreateBookRequest br = new CreateBookRequest();
+        br.setTitle(TITLE);
+        br.setDescription(DESCRIPTION);
+        br.setAuthorId(AUTHOR_ID);
+        br.setReleaseDate(RELEASE_DATE);
+
+        when(authorRepository.findById(br.getAuthorId())).thenReturn(Optional.ofNullable(author));
+
+        Book actualBook = bookMapper.mapCreateRequestToEntity(br);
+        assertEquals(book.getTitle(), actualBook.getTitle());
+        assertEquals(book.getDescription(), actualBook.getDescription());
+        assertEquals(book.getReleaseDate(), actualBook.getReleaseDate());
+        assertEquals(book.getAuthor(), actualBook.getAuthor());
+    }
+
+    @Test
+    void mapCreateRequestToEntityThrowsExceptionWhenAuthorIdNull() {
+        assertThrows(RuntimeException.class,
+                () -> authorRepository.findById(null).orElseThrow(RuntimeException::new));
+    }
```

Comment:
you don't test here anything from bookMapper class by that method, since you're calling authorRepository, you do test that repository.. 
The method name is correct, but do next adjustments
```suggestion
    void mapCreateRequestToEntityThrowsExceptionWhenAuthorIdNull() {
    bookRequest.setAuthorId(1L)
        when(authorRepository.findById(bookRequest.getAuthorId()).thenReturn(Optional.emtpy))
        assertThrows(RuntimeException.class,
                () ->  bookMapper.mapCreateRequestToEntity(bookRequest);
    }
```